static/templates/dialog_change_password.hbs

```diff
@@ -15,5 +15,6 @@
         <div class="progress inline-block" id="pw_strength">
             <div class="bar bar-danger fade" style="width: 10%;"></div>
         </div>
+        <p><b>Note:</b> This will log you out of all devices</p>
```

Comment:
I think this should be tagged for translation, like the other strings here

Reply:
Oh, right, tagged it for translation.